### PR TITLE
Styling 

### PR DESCRIPTION
**Pull request by demesameneshoa**
## In this PR I executed 🔢 
- [x] Styling Body, Navbar and Content Wrapper
- [x] Styling Book Cards and Percentage Details
- [x] Styling New Book Form
- [x] Refactoring Components to suit the styling

### CODE REVIEW

File: src/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+  align-items: center;
+}
+
+.Bookstore-CMS,
+.categories-page {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  padding-left: 1rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
```
Comment (Tufahel) [inline src/styles/styles.css:43]: - Nice use of `font-family` this way. I didn't know it. Have learned new information. Thanks. 👍
- Happy Coding 😄

File: src/styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+#author-input {
+  background: #ffff;
+  border: solid 1px #e8e8e8;
+  width: 35%;
+  height: 2.8rem;
+  font-size: 1rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  margin-left: 2rem;
+  margin-top: 2rem;
+}
```
Comment (Tufahel) [inline src/styles/styles.css]: - [x] Kindly give some padding around `BOOK TITLE` and `Book Author` to match the given design.

- [x] You missed a `down arrow` in the `Book Author` input tag. Kindly fix this issue also to match the given design.

image

File: src/styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+.chapter-num {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
```
Comment (Tufahel) [inline src/styles/styles.css]: - [x] Kindly fix the spacing/padding/margin issue between `Current Chapter`, `Chapter 17` and the `Button Update Progress` to match the given design.

image

File: src/styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,306 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700;800;900&display=swap');
+
+body {
+  width: 100%;
+  height: auto;
+  background-color: #f5f6fa;
+  text-decoration: none;
+}
```
Comment (Tufahel) [inline src/styles/styles.css:9]: image

- [x] You can see the contents are not perfectly aligned to the `left side` and the `right side`. Kindly fix this issue to match the given design guideline.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @demesameneshoa :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Follow the correct Git flow.
✔️ App is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/styles.css (hunk 1 above)
Comment (demesameneshoa) [reply src/styles/styles.css:43]: @Tufahel  Thanks for your feedback

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @demesameneshoa 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**